geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/conf/ConfigLoader.scala

```diff
@@ -0,0 +1,59 @@
+/***********************************************************************
+  * Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+  * All rights reserved. This program and the accompanying materials
+  * are made available under the terms of the Apache License, Version 2.0
+  * which accompanies this distribution and is available at
+  * http://www.opensource.org/licenses/apache2.0.php.
+  *************************************************************************/
+
+package org.locationtech.geomesa.utils.conf
+
+import scala.xml.XML
+import java.io.FileNotFoundException
+
+import com.typesafe.scalalogging.LazyLogging
+
+object ConfigLoader extends LazyLogging {
+  val ACCUMULO_HOME = "ACCUMULO_HOME"
+  val KAFKA_HOME = "KAFKA_HOME"
+  val CONFIG_PATH = "/conf/geomesa-site.xml"
+
+  def init(): Unit = {
+    if (! Option(System.getenv(ACCUMULO_HOME)).getOrElse("").isEmpty) {
+      this.init(ACCUMULO_HOME)
+    } else if (! Option(System.getenv(KAFKA_HOME)).getOrElse("").isEmpty) {
+      this.init(KAFKA_HOME)
+    } else {
+      logger.warn("Unable to locate GeoMesa config file.")
+    }
+  }
+
+  def init(envName: String): Unit = {
+    if (! Option(System.getProperty("geomesa.config.file")).getOrElse("").isEmpty){
+      this.loadConfig(System.getProperty("geomesa.config.file"))
```

Comment:
I would think the order for finding the file should be:
1. explicit system property
2. check the current classpath
3. check for GEOMESA_ACCUMULO_HOME, GEOMESA_KAKFA_HOME
@jahhulbert-ccri what do you think?